### PR TITLE
 Prevent Following Redirect (#615)

### PR DESCRIPTION
**Pull request by fab33**
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Proposed changes
 Prevent Following Redirect #615

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: nodes/core/io/21-httprequest.html [hunk 1]
```diff
@@ -88,6 +88,8 @@ <h3>Inputs</h3>
         <dt class="optional">rejectUnauthorized</dt>
         <dd>If set to <code>true</code>, allows requests to be made to https sites that use
             self signed certificates.</dd>
+        <dt class="optional">followRedirects</dt>
+        <dd>If set to <code>false</code> prevent following Redirect (HTTP 301).<ode>true</code> by default</dd>
```
Comment (dceejay) [inline nodes/core/io/21-httprequest.html]: what is this <ode> tag ;-)

File: nodes/core/io/21-httprequest.js [hunk 2]
```diff
@@ -108,6 +108,8 @@ module.exports = function(RED) {
                     }
                 }
             }
+            if (msg.hasOwnProperty('followRedirects'))
```
Comment (dceejay) [inline nodes/core/io/21-httprequest.js]: we like { } :-)

**Comment by fab33** [comment]
Sorry, first PR for me ;)

**Comment by dceejay** [comment]
Thanks for your first PR !

**Pull request merged**